tests/test_pydbg.py

```diff
@@ -17,8 +19,8 @@ def test_variables():
     strType3 = "2"
     strType4 = "3.4"
 
-    out = io.StringIO()
-    with redirect_stdout(out):
+    out1 = io.StringIO()
```

Comment:
why the rename?

Reply:
Will revert the rename. Initially I was adding more tests (for stdout), but had to drop them.